sql/init.sql

```diff
@@ -1,7 +1,5 @@
-CREATE DATABASE movies_db;
-
 -- Actors table
-CREATE TABLE public.actors
```

Comment:
If not public, in which schema are tables created? I don't see any schema creation statement